Formula/vdirsyncer.rb

```diff
@@ -1,8 +1,9 @@
+# coding: utf-8
```

Comment:
No problem! Sometimes I wonder if we're controlling the machines or the machines are controlling us 😉.